packages/mgt-components/src/components/mgt-file-list/mgt-file-list.ts

```diff
@@ -148,31 +148,6 @@ export class MgtFileList extends MgtTemplatedComponent {
     return getSvg(SvgIcon.Files);
   }
 
-  /**
```

Comment:
By making the mgt-file-list extend BasePersonCardSection we don't need to have explicit implementations here

Reply:
And re-thinking this we should have these methods on the interface that the section components implement